project2/styles.css

```diff
@@ -0,0 +1,124 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    background-color: rgb(100, 100, 100);
+    color: rgb(255, 255, 255);
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 12px;
+}
+
+a {
+    text-decoration: none;
+    color: rgb(255, 255, 255);
+}
+
+h2 {
+    font-family: 'Catamaran', sans-serif;
+    font-size: 30px;
+    font-weight: bold;
+    line-height: 26px;
+}
+
+/*===== class  =====*/
+
+.button {
+    background-color: rgb(246, 197, 24);
+    border-radius: 4px;
+    box-shadow: 7px 7px rgba(0, 0, 0, 0.2);
+    color: rgb(255, 255, 255);
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 12px;
+    display: inline-block;
+    font-weight: bold;
+    padding: 11px;
+    position: relative;
+    text-decoration: none;
```

Comment:
You already removed `text-decoration` on the `a` element, so you can delete this line.